index.bs

```diff
@@ -974,7 +982,8 @@ method is invoked, the user agent MUST:
 
     1. [=set/Append=] |authenticator| to |issuedRequests|.
 
-1. Execute the following steps [=in parallel=]. The [=task source=] for these [=tasks=] is the [=dom manipulation task source=].
+1. Execute the following steps [=in parallel=] but abort if the steps are [=terminated=]. The [=task source=] for these [=tasks=]
```

Comment:
"Abort if the steps are terminated" isn't quite the right thing to say here. First, even Fetch says, ["if the ongoing fetch is terminated"](https://fetch.spec.whatwg.org/#concept-main-fetch), rather than saying steps might be terminated.

Second, aborting steps just stops running them. For WebAuthn, we need to run authenticatorCancel at the very least, which seems to mean you should use https://dom.spec.whatwg.org/#abortsignal-add. See the "Add the following abort steps" line in https://fetch.spec.whatwg.org/#dom-global-fetch for an example.

Reply:
plus we need to get PR #665 merged-to-master, then this branch merged-from-master in order to be working on the latest state of this alg.

Reply:
agree w/@jyasskin that we may need to employ  https://dom.spec.whatwg.org/#abortsignal-add techniques similar to how [fetch](https://fetch.spec.whatwg.org/#fetch-method) handles aborting.  It also seems that fetch has a notion of "[An ongoing fetch can be terminated
with flag aborted](https://fetch.spec.whatwg.org/#concept-fetch-terminate)" which is local to the fetch spec, and not part of [the DOM's "aborting ongoing activities"](https://dom.spec.whatwg.org/#aborting-ongoing-activities). I wonder whether we might need to implement something similar.